File: Cpp/389_Find_the_Difference.cpp
class Solution {
public:
    char findTheDifference(string s, string t) {
        char result = 0;
        for(auto& ch : s) {
            result ^= ch;
        }
        for(auto& ch : t){
            result ^= ch;
        }
        
        return result;
    }
};

File: Kotlin/389_Find_the_Difference.kt
class Solution {
    fun findTheDifference(s: String, t: String): Char {
        var result = 0
        
        s.forEach{ ch ->
            result = result xor ch.toInt()
        }
        
        t.forEach{ ch ->
            result = result xor ch.toInt()
        }
        
        return result.toChar()
    }
}